packages/testing/pyproject.toml

```diff
@@ -51,6 +51,7 @@ dependencies = [
     "joblib>=1.4.2",
     "ckzg>=2.1.3,<3",
     "tenacity>=9.0.0,<10",
+    "ruff==0.13.2",
```

Comment:
This is a bit crazy, but currently `gentest` runs `ruff`. It didn't really matter when we all just worked in dev mode with `uv sync --all-extras`, but this is now required to fix our package.

Perhaps there should be a `gentest` extra where this dependency is added?

Reply:
> Perhaps there should be a `gentest` extra which only adds this dependency?

Yeah, that makes sense. Could even make the formatting step optional, and only run if `ruff` is available?

Reply:
I agree, I think the optional formatting if available makes sense. In most cases it will be available because anyone developing will have `dev` deps installed. If it isn't, you have to lint your code before merging anyway so would just be an extra step.